Source/VCGeneration/Counterexample.cs

```diff
@@ -81,8 +83,6 @@ void ObjectInvariant()
     public Model Model;
     public VC.ModelViewInfo MvInfo;
     public ProverContext Context;
-    public string OriginalRequestId;
-    public string RequestId;
```

Comment:
The reason I wanted to remove the fields is that they enable removing one parameter from `VerifyImplementationWithLargeStackScheduler`, which is called by the new code in `ImplementationTask`, and which would otherwise pass in `null` for this parameter since it doesn't have a requestId to pass in. There's other parameters of `VerifyImplementationWithLargeStackScheduler` like `ErrorReporterDelegate er` that I think are redundant, but I've left those alone for now.

I checked that Corral and GPUVerify do not use the fields that I've removed, and AFAIK there aren't other consumers of the Boogie's C# API.